content/source/docs/enterprise-beta/api/policy-checks.html.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: enterprise2
+page_title: "Policy Checks - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-policy-checks"
+---
+
+# Policy Checks API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## List policy checks
+
+This endpoint lists the policy checks in a run
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /runs/:run_id/policy-checks |
+
+### Parameters
+
+- `run_id` (`string: <required>`) - specifies the run ID for which to list policy checks
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
```

Comment:
You don't need to supply the content-type on a GET request.

Reply:
It's fixed, but not sure why it isn't showing here.

Reply:
Weird - I swear I made this comment on a GET, but this is the POST for overriding. 🤷‍♂️